demo/chbench/mzcompose.yml

```diff
@@ -253,9 +253,20 @@ mzconduct:
     load-test:
       steps:
       - step: workflow
-        workflow: nightly
+        workflow: bring-up-source-data
+      # start the dashboard before the load data get started so that we have data from
```

Comment:
Should we do this in `nightly` as well?

Reply:
nightly will have an external prometheus discovering that this is happening by polling the ec2 api, so it will find the load tests automatically. (It runs its own dashboard and prometheus.)